grade.go

```diff
@@ -363,7 +377,7 @@ func main() {
 	input := parseInFile(inFile)
 
 	expected := getFile(workDir + "/.spec/out.txt")
-	fmt.Println(expected)
+	fmt.Println("Expected Output: ", expected)
```

Comment:
add a newline after this since this will offset output

Reply:
Going to add that newline on the next PR